web_development_101/project_js_jquery.md

```diff
@@ -39,6 +39,7 @@ If you get totally stuck, solutions from other students are listed below.  I'd e
 *Submit a link below to the github repo with your files in it by using a pull request.  See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.  Please include your partner's github handle somewhere in the description if you had one and they would like attribution.*
 
 * Add your solution below this line!
+* [adsy430's Solution](https://github.com/adampal/etcha-sketch) | [View in browser] (https://adampal.github.io/etcha-sketch/)
```

Comment:
You need to remove the space between 'view in browser' and the ( ) with the link in order to make the view in browser a link.